### DISCUSSION TITLE
Don't shift the "+" and "Display options menu" buttons around when dynamic panels are enabled

### DISCUSSION THREAD

**Discussion by jphalip** [standardnotes/forum#3081] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**

Yes

**Is your feature request related to a problem? Please describe.**

If you enable dynamic panels, the note list panel gets reduced and then expands out from left to right when you hover it. This is fine, except that the "+" button (to create a new note) also shifts right while the list panel gets expanded. This is a bit jarring/frustrating when you try to aim for the button with your mouse and have to wait for the expansion to finish, and then have to move your mouse cursor again to finally be able to click the button.

**Describe the solution you'd like**

I would suggest that, when dynamic panels are enabled, both the "+" button and "Display options menu" button get pinned down where they are and don't shift when the list panel gets expanded.

**Comment by zaernch** [comment]
+1 for this request.

I notice that there is this option:

> Account Preference -> General -> Advanced Options -> Dynamic Panels -> Uninstall

Would that get rid of the annoying button shifting?

**Comment by effieeee** [comment]
Hi @zaernch, I'm afraid that would remove Dynamic Panels entirely, so it doesn't quite solve the problem described 😞
